File: 04_sorting/04_03_insertion_sort/insertion_sort.js
const insertionSort = (arr = []) => {
    for (let index = 0; index < arr.length; index++) {
        for (let j_index = index; j_index > 0; j_index--) {
            if (arr[j_index] < arr[j_index - 1]) {
                [arr[j_index], arr[j_index - 1]] = [arr[j_index - 1], arr[j_index]];
            }
        }
    }
}

const array = [13, 46, 24, 52, 20, 9]

console.log("Before:-");
console.log(array);
insertionSort(array)
console.log("After:-");
console.log(array);

File: 04_sorting/04_03_insertion_sort/insertion_sort.py
def insertionSort(arr: list):
    for i in range(len(arr)):
        for j in range(i, 0, -1):
            if arr[j] < arr[j-1]:
                arr[j], arr[j-1] = arr[j-1], arr[j]
                
                
elements = [13, 46, 24, 52, 20, 9]
print("Before:-")
print(elements)
insertionSort(elements)
print("After:-")
print(elements)